rest-api-spec/src/main/resources/rest-api-spec/api/_global.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Parameters that are accepted by all API endpoints.",
+  "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html",
+  "params": {
+    "pretty": {
+      "type": "boolean",
+      "description": "Pretty format the returned JSON response.",
+      "default": false
+    },
+    "human": {
+      "type": "boolean",
+      "description": "Return human readable values for statistics.",
+      "default": true
+    },
+    "error_trace": {
+      "type": "boolean",
+      "description": "Include the stack trace of returned errors.",
+      "default": false
+    },
+    "source": {
+      "type": "string",
+      "description": "The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests."
+    },
+    "filter_path": {
```

Comment:
@gmarz I think every client supports it, or at least every client that runs the yaml tests. `ignore` may indicate some error status codes for which a particular request shouldn't fail. For instance get api can return 404 with a normal get body, which is not an exception although it doesn't hold the document. We use it in some of the yaml tests.